chain/network/src/routing/routing.rs

```diff
@@ -260,8 +260,8 @@ impl RoutingTableView {
 
     pub fn remove_edges(&mut self, edges: &Vec<Edge>) {
         for edge in edges.iter() {
-            assert!(edge.key.0 == self.my_peer_id || edge.key.1 == self.my_peer_id);
-            let key = (edge.key.0.clone(), edge.key.1.clone());
+            assert!(edge.key().0 == self.my_peer_id || edge.key().1 == self.my_peer_id);
+            let key = (edge.key().0.clone(), edge.key().1.clone());
```

Comment:
same, can just use `edge.key()`

Reply:
Thanks, fixed. I found a few other places like that as well.